### PR TITLE
buttons are alright!

### PR DESCRIPTION
**Pull request by Anhelina4**
buttons are alright

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,5 +19,8 @@
   "homepage": "https://github.com/jazzkrop/booking#readme",
   "dependencies": {
     "bootstrap": "^5.0.2"
+  },
+  "devDependencies": {
+    "sass": "^1.37.5"
```
Comment (Olexandr25) [inline package.json]: Why did you add "sass" version"

Comment (jazzkrop) [inline package.json]: why did you add this package?

**Review by jazzkrop — COMMENTED** [review]
all right but why you added sass?
